components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -1644,4 +1644,37 @@ private static String hexify(byte bytes[]) {
         return buf.toString();
     }
 
+    public static ArrayList<String> getEssentialClaims(String essentialClaims, String claimType) {
```

Comment:
better to use interfaces i.e. List<String> instead of ArrayList<String> in method signatures.

Reply:
Fixed in master - https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java#L2375